lib/enums/ScoreEvaluateAction.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
Please break into several different objects by type.
Then export them with names. (A mock for namespace)

Reply:
I see 0s' and 1s', not just for a single value.